From 5e4e65748f8558a7ed0bbd3c56832f89357efac8 Mon Sep 17 00:00:00 2001
From: Yong Chen <yong.a.chen@intel.com>
Date: Mon, 9 Sep 2013 14:04:48 +0800
Subject: Add Version.h for RS Vectorizer

BZ: 137008

Version.h defines LLVM_VERSION which is used by RS Vectorizer.

Category: feature differentiation
Domain: AOSP-RenderScript
Origin: internal
Upstream-Candidate: no, proprietary

Change-Id: If856a8ad7790894b5b06181f0b68c53735c8ba86
Orig-MCG-Change-Id: I18dec55647c2b9b60fe9cdab9ffc6c976d40f7fa
Signed-off-by: Yong Chen <yong.a.chen@intel.com>
Reviewed-on: http://android.intel.com:8080/130787
Tested-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/llvm/Version.h |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)
 create mode 100644 include/llvm/Version.h

diff --git a/include/llvm/Version.h b/include/llvm/Version.h
new file mode 100644
index 0000000..6c4bf06
--- /dev/null
+++ b/include/llvm/Version.h
@@ -0,0 +1,6 @@
+#ifndef __VERSION_H
+#define __VERSION_H
+
+#define LLVM_VERSION 3300
+
+#endif
-- 
1.7.4.1

